db/re/skill_db.yml

```diff
@@ -6305,7 +6305,7 @@ Body:
     Type: Weapon
     TargetType: Attack
     DamageFlags:
-      IgnoreFlee: true
+      IgnoreFlee: false
```

Comment:
Do you have source for this change? I was looking back at the changelog but I don't see it stating anything about a change with flee.

Reply:
@aleos89  I think I read that somewhere I just can’t remember where. I’ll try to dig that info but it’s correct that they dropped the perfect hit for Backstab cause I’m sure I read that.

Reply:
@aleos89  There's a increase hit bonus according to level of skill based on the translate kRO patch notes. Not sure why it would give hit bonus if it ignores flee